### DISCUSSION TITLE
No Support for with() eager loading

### DISCUSSION THREAD

**Discussion by zoomyboy** [larastan/larastan#1164] [Q&A] [answered]
Hello,

If I rename a with('...') Eloquent call to some relation that doesnt exist on the model, Larastan is not catching this, right? Can I fix this somehow?

**Comment by canvural** [comment, marked as answer]
Hi,

This was missing from the rule. I added support for it in https://github.com/nunomaduro/larastan/pull/1175

Thanks for the idea!
